README.md

```diff
@@ -30,14 +30,14 @@ the Extraction tool binary (i.e., ExtractionTool_deploy.jar) and its run script.
 
 **Step 3:** Create a folder for output files and run the Extraction tool:
 
-Option 1: basic run:
 ```bash
 ./run-td-extract.sh -j <terajdbc4.jar> --db-address <database address> --output <output path> --db-user <db user>
 ```
-Option 2: compatible with the full set of optional parameters (See **Optional parameters** section):
```

Comment:
Let use keep the reference to running `dwh-assessment-execution-tool td-extract` (but correct the missing `td-extract`). The `run-td-extract` script is meant for the straight forward use cases. So, we should keep the reference to `dwh-assessment-execution-tool` for more sophisticated use cases.

Reply:
Agree. Done.